model.js

```diff
@@ -5,3 +5,41 @@ exports.selectTopics = () => {
     return result.rows;
   });
 };
+
+exports.selectArticleById = (articleId) => {
+  return connection
+    .query(
+      `
+      SELECT * FROM articles
+      WHERE article_id = $1
+        `,
+      [articleId]
+    )
+    .then((result) => {
+      if (result.rowCount !== 0) return result.rows[0];
+      return Promise.reject({
+        status: 404,
+        message: "path not found",
+      });
+    });
+};
+
+exports.selectArticleWithNewVotes = (article_id, inc_votes) => {
+  const patchArr = [];
+  patchArr.push(article_id, inc_votes);
```

Comment:
Seem like the push is unnecessary here when you could have just put the id and votes in the array in the first place.